kubernetes.yaml

```diff
@@ -46,6 +46,11 @@ spec:
         ports:
         - name: prom-metrics
           containerPort: 9001
+        env:
```

Comment:
Is the time range between which the preemptibles get killed configurable as well?

Reply:
It was not, I'm going to make it configurable later today

Reply:
Okay, that can be done in a separate PR than. I'll merge this one.